Formula/s6.rb

```diff
@@ -0,0 +1,30 @@
+class S6 < Formula
+  desc "Small & secure supervision software suite."
+  homepage "http://skarnet.org/software/s6/"
+  url "http://skarnet.org/software/s6/s6-2.3.0.0.tar.gz"
+  sha256 "49c9a0476d93c6279c776ca5ae9c49b3fd39c356eda674828aebf78f8488e8e6"
+  head "git://git.skarnet.org/s6"
+
+  depends_on "skalibs"
+  depends_on "execline"
+
+  def install
+    system "./configure",
+      "--prefix=#{prefix}",
+      "--with-sysdeps=#{Formula["skalibs"].opt_prefix}/lib/skalibs/sysdeps",
```

Comment:
I appreciate you likely just mimicked the lines from `execline` but they're wrong there too, sadly 😄.